helm/observability-operator/values.yaml

```diff
@@ -9,6 +9,7 @@ image:
 
 managementCluster:
   baseDomain: domain
+  clusterIssuer: letsencrypt-giantswarm
```

Comment:
I have seen `selfsigned-issuer` being used before I am confused now :confused:

Reply:
Both exists and we need selfsigned if we are in the cluster, letencrypt if we are outside (iike an ingress)

Reply:
```suggestion
  clusterIssuer: selfsigned-issuer
```